ui/app/pages/permissions-connect/redirect/permissions-redirect.component.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react'
+import PropTypes from 'prop-types'
+import IconWithFallBack from '../../../components/ui/icon-with-fallback'
+import { I18nContext } from '../../../contexts/i18n'
+
+export default function PermissionsRedirect ({ domainMetadata }) {
+
+  const t = useContext(I18nContext)
+
+  return (
+    <div className="page-container permissions-redirect-container">
```

Comment:
Nit: These classes aren't the most BEM-like. `-container` sounds like it should be an element, not a block (because it isn't really a "thing" on its own), and there seems no reason to split these components between two different blocks (`permission-result` and `permission-redirect-container`).

I'm guessing some of this was pre-existing though.

Reply:
Yeah, a lot of it was pre-existing. Not sure if the names are an improvement or not, but the HTML and CSS have both been simplified.

Reply:
14a5e77